main.go

```diff
@@ -72,6 +88,54 @@ func ToEther(o *big.Int) *big.Float {
 	return pul
 }
 
+func CallContractFunction(contractAddress string, abiString string, functionName string) (string, error) {
+	// Parse ABI
+	contractABI, err := abi.JSON(strings.NewReader(abiString))
+	if err != nil {
+		return "", fmt.Errorf("failed to parse ABI: %v", err)
+	}
+
+	// Pack the function call
+	data, err := contractABI.Pack(functionName)
```

Comment:
It may be worth adding parameters to support functions that require them.

```suggestion
	data, err := contractABI.Pack(functionName, params...)
```

Reply:
[1d83a66](https://github.com/mezo-org/balance-exporter/pull/1/commits/1d83a661ebe192098ed0e2cd393049023572ceea)